### PR TITLE
Project Settings > details

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Refactoring the project settings pages.  This is is to handle **only** project details.  Updates the way the data is fetch and updated to us redux.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

N/A

### CODE REVIEW

File: src/main/webapp/resources/js/apis/projects/settings.js [hunk 1]
```diff
@@ -47,7 +36,7 @@ export const fetchProcessingCoverage = async (projectId) =>
  * @param {object} coverage - updated coverage for the project
  * @returns {Promise<*>}
  */
-export const updateProcessingCoverage = async (projectId, coverage) => {
+export const putProcessingCoverage = async (projectId, coverage) => {
```
[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/resources/js/apis/projects/settings.js:39]: There's a few other places where you use `putXX`. It just feels dirty to me. So if you could update those as well that'd be great

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Overall looks great and works as expected. Just a few changes for now

File: src/main/webapp/resources/js/apis/projects/settings.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/apis/projects/settings.js:39]: Updated in 2d2bfd6eed0ee17428b9087e52e0e4e00d1dd8d2

File: src/main/webapp/resources/js/pages/projects/settings/components/processing/AnalysisTemplates.jsx [hunk 2]
```diff
@@ -16,24 +18,26 @@ import { IconRemove } from "../../../components/icons/Icons";
...
   }, []);
 
-  const removeAutomatedPipeline = (template) => {
-    deleteAnalysisTemplateForProject(template.id, projectId).then((message) => {
-      notification.success({ message });
-      const filteredTemplates = templates.filter((t) => t.id !== template.id);
-      setTemplates(filteredTemplates);
-    });
-  };
+  const removeAutomatedPipeline = (template) =>
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/settings/components/processing/AnalysisTemplates.jsx]: Updated in 5352a80

File: src/main/webapp/resources/js/pages/projects/redux/pipelinesSlice.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  }
+);
+
+/**
+ * Delete an automated analysis pipeline from the project.
+ * @type {AsyncThunk<{message: *, templateId: *}, {readonly templateId?: *, readonly projectId?: *}, {}>}
+ */
+export const deletePipeline = createAsyncThunk(
+  `pipelines/deletePipeline`,
+  async ({ templateId, projectId }) => {
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/redux/pipelinesSlice.js]: Updated in 5352a80

File: src/main/webapp/resources/js/pages/projects/settings/components/processing/ProcessingPriorities.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+ *
+ * @param {number} projectId - project identifier
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function ProcessingPriorities({ projectId }) {
+  const dispatch = useDispatch();
+  const { priority } = useSelector((state) => state.project);
+  const PRIORITIES = [
+    { value: "LOW", label: i18n("AnalysisDetailsPriority.LOW") },
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/settings/components/processing/ProcessingPriorities.jsx:18]: Don;t want to change more now.  Maybe a future idea

[the remark this answers is not in the document]
Comment (deepsidhu85) [reply src/main/webapp/resources/js/pages/projects/settings/components/processing/ProcessingPriorities.jsx:18]: I will give you a pass on this one :)

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**